source/LogbookPanel.cpp

```diff
@@ -155,6 +175,10 @@ void LogbookPanel::Draw()
 			pos.Y() += wrap.Height() + GAP;
 		}
 	}
+	else if(!selectedDate && selectedName == STORYLINES)
+	{
+		ShowStorylines(font, medium, dim, textOffset, pos);
+	}
```

Comment:
```suggestion
		ShowStorylines(font, medium, dim, textOffset, pos);
```
No need for braces when its contained in one block (in this case one line)